resources/knative-serving-crds/README.md

```diff
@@ -0,0 +1,8 @@
+# Knative Serving CRDs
+
+## Overview
+
+>**NOTE**: This is an experimental module.
+This chart includes [knative-serving-crds](https://github.com/knative/docs/tree/master/docs/serving) release files.
```

Comment:
```suggestion

This chart includes [knative-serving-crds](https://github.com/knative/docs/tree/master/docs/serving) release files.
```
Add a line, otherwise this part is included in the note.